docs/supported-libraries.md

```diff
@@ -17,111 +17,128 @@ or [contributing](../CONTRIBUTING.md).
+| [gRPC](https://github.com/grpc/grpc-java)                                                                                         | 1.6+                          | [RPC Server Spans], [RPC Client Spans], [RPC Server Metrics], [RPC Client Metrics]     |
+| [Guava ListenableFuture](https://guava.dev/releases/snapshot/api/docs/com/google/common/util/concurrent/ListenableFuture.html)    | 10.0+                         | TODO                                                                                   |
+| [GWT](http://www.gwtproject.org/)                                                                                                 | 2.0+                          | TODO                                                                                   |
+| [Hibernate](https://github.com/hibernate/hibernate-orm)                                                                           | 3.3+ (not including 6.x yet)  | TODO                                                                                   |
+| [HikariCP](https://github.com/brettwooldridge/HikariCP)                                                                           | 3.0+                          | TODO                                                                                   |
+| [HttpURLConnection](https://docs.oracle.com/en/java/javase/11/docs/api/java.base/java/net/HttpURLConnection.html)                 | Java 8+                       | TODO                                                                                   |
+| [Hystrix](https://github.com/Netflix/Hystrix)                                                                                     | 1.4+                          | TODO                                                                                   |
+| [Java Executors](https://docs.oracle.com/javase/8/docs/api/java/util/concurrent/Executor.html)                                    | Java 8+                       | TODO                                                                                   |
+| [Java Http Client](https://docs.oracle.com/en/java/javase/11/docs/api/java.net.http/java/net/http/package-summary.html)           | Java 11+                      | TODO                                                                                   |
+| [java.util.logging](https://docs.oracle.com/javase/8/docs/api/java/util/logging/package-summary.html)                             | Java 8+                       | TODO                                                                                   |
+| [Java Platform](https://docs.oracle.com/javase/8/docs/api/java/lang/management/ManagementFactory.html)                            | Java 8+                       | [JVM Runtime Metrics]                                                                  |
```

Comment:
This was also missing. I know its also mentioned on the [standalone library](https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/main/docs/standalone-library-instrumentation.md) doc, but its worth including it here to indicate that the JVM runtime semantic conventions are implemented.

On a related note, we may want to merge the standalone library doc page with this. Could add a new column indicating whether the library has library instrumentation, and if yes, include a link to the module. Or find a way to reference the maven coordinates?

Reply:
> Could add a new column indicating whether the library has library instrumentation, and if yes, include a link to the module.

👍